prod.env

```diff
@@ -0,0 +1,3 @@
+QB_CONSUMER_KEY=qyprdSHw9jL4KyKToh3PYK9aojXcyO
+QB_CONSUMER_SECRET=kxknADzRuMHAedKGKBe5JCHhte72c36YLAbcs41F
+VIRTUAL_HOST=quickbooks_integration.flowlink.io
```

Comment:
This kind of info should NEVER be on a repository. You could leave it as "QB_CONSUMER_KEY=<put key here>" for reference, but the actual values should be stored on Lastpass or anywhere else -- never on the repo.